clinic/clinic-user-guide-for-tiup.md

```diff
@@ -5,9 +5,9 @@ summary: Learn how to use the PingCAP Clinic Diagnostic Service to troubleshoot
 
 # Use PingCAP Clinic
 
-For TiDB clusters and DM clusters deployed using TiUP, you can use PingCAP Clinic Diagnostic Service (PingCAP Clinic) to troubleshoot cluster problems remotely and perform a quick check on a cluster status locally using a Clinic diagnostic tool Diag (Diag) and a Clinic Server cloud service (Clinic Server).
+For TiDB clusters and DM clusters deployed using TiUP, you can use PingCAP Clinic Diagnostic Service (PingCAP Clinic) to troubleshoot cluster problems remotely and perform a quick check on a cluster status locally using Diag client (Diag) and a [Clinic Server China](https://clinic.pingcap.com.cn)(Clinic Server). For details about Diag and Clinic server, see [PingCAP Clinic components](/clinic/clinic-introduction.md)。
```

Comment:
@qqqdan @qiancai Added the last sentence for global users in case some users will be curious about the difference between Clinic Server China and Clinic Server. PTAL, thanks!